src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java

```diff
@@ -79,11 +80,11 @@ public void tearDown() {
   @Test
   public void doPost_urlAlreadyInDatabase_shouldReturnLecture() throws IOException {
     when(request.getParameter(LINK_INPUT)).thenReturn(TEST_LINK);
-    datastoreService.put(servlet.createLectureEntity(request));
+    datastoreService.put(servlet.getLectureEntity(request));
```

Comment:
Now that createEntity is public I would say you should try to not make getLectureEntity @VisibleForTesting. If you want we can talk about how to achieve this.

Reply:
Made the fix